ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -65,16 +65,21 @@ Listen <%= addr_port %>
 <VirtualHost *:<%= @port %>>
   <%- if @servername -%>
   ServerName <%= @servername %>
+  <%- end -%>
 
-  ErrorLog  "<%= @logroot %>/<%= @servername %>_error<%= "_ssl" if @ssl %>.log"
-  CustomLog "<%= @logroot %>/<%= @servername %>_access<%= "_ssl" if @ssl %>.log" combined
+  ErrorLog  "<%= @errorlog %>"
```

Comment:
This is the test failure diff:

```
       +  ErrorLog  "logs/error.log"
       +  CustomLog "logs/access.log" combined
       +
       +
```

The `ErrorLog` and `CustomLog` used to only show when `@servername` was present but now they can exist without @servername.  So you will need to update the test files, and the unfortunate part is it also updates some SHA1 sum tests which are sort of a pain to update.  Let me know if you need help getting the tests to pass.

Reply:
Yes, that's what I was thinking too. Updated the relevant test fixtures and the default checksum. Unit tests are all passing now. Hope I'm not missing anything.